amazon-link-engine/trunk/amazon-link-engine.php

```diff
@@ -147,7 +149,7 @@ function genius_ale() {
 }
 
 if (!is_admin()) {
-  add_action('wp_head', 'genius_ale');
+  add_action('wp_head', 'genius_ale', 9999);
```

Comment:
Cool, does this just change the position of the snippet in the HTML? Did this solve a problem for one of your sites?